lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "4.18.0"
+  "version": "4.18.1-rc.1"
```

Comment:
Version update is managed by Ship.js so you don't need to manually set your new version (same for `packages/client-common/src/version.ts` and `packages/recommend/package.json`).

I'm not sure of the way to go if you want to release a RC version of the recommend client. One simple way is to stay in this feature branch and publish manually. This way you'll only release `@algolia/recommend@4.18.1-rc.1` and not all of the other packages in the same monorepo. Probably need a validation from the API clients owners.

Reply:
Yes apologies, this was not meant to be for the actual PR. 

It was just to check if my CodeSandbox was pulling the right version. I'll revert the change before setting the PR as open.